db/migrate/20160514215652_merge_first_name_and_last_name_on_users.rb

```diff
@@ -0,0 +1,8 @@
+class MergeFirstNameAndLastNameOnUsers < ActiveRecord::Migration
```

Comment:
FYI this will destroy existing name data. Take a look at http://stackoverflow.com/questions/25411221/combine-columns-in-a-rails-migration